### PR TITLE
add examples/point

### PR DESCRIPTION
**Pull request by mw66**
Hi,

I just add the example code from README.md to examples/point (and fixed build flag -lm).

Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,13 @@ Minimum Supported Rust Version: `1.56.0`
 
 ## Quickstart
 
+Please try:
```
Comment (danielhenrymantilla) [inline README.md:34]:
```suggestion
You may try working with the `examples/point` example embedded in the repo:
```

File: examples/point/Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+point:
+	cargo build # --release
+	cargo test --features c-headers -- generate_headers
+	cc main.c -o main -L ../../target/debug -l point -l pthread -l dl -lm
```
Comment (danielhenrymantilla) [inline examples/point/Makefile:5]:
```suggestion
main: main.c rustlib rust_points.h
	cc $< -o $@ -L ../../target/debug -l point -l pthread -l dl -lm
	
.PHONY: rustlib
rustlib:
	cargo build # --release

rust_points.h:
	cargo test --features c-headers -- generate_headers
```

File: README.md [hunk 3]
```diff
@@ -31,6 +31,13 @@ Minimum Supported Rust Version: `1.56.0`
...
+```
+$ cd examples/point
```
Comment (danielhenrymantilla) [inline README.md:36]:
````suggestion
```bash
git clone https://github.com/getditto/safer_ffi && cd safer_ffi
cd examples/point
````

**Review by danielhenrymantilla — CHANGES_REQUESTED** [review]
Thanks for PR! The idea looks very sensible to me; if you can address the nits I've pointed out I'll gladly get it merged 🙂

**Comment by mw66** [comment]
Thanks, change committed.

**Review by danielhenrymantilla — APPROVED** [review]

**Pull request merged**